packages/cli/src/ui/components/shared/text-buffer.test.ts

```diff
@@ -2241,51 +2269,64 @@ describe('Transformation Utilities', () => {
     });
 
     it('should handle different image extensions', () => {
-      expect(getTransformedImagePath('@test.jpg')).toBe('[Image @test.jpg]');
-      expect(getTransformedImagePath('@test.jpeg')).toBe('[Image @test.jpeg]');
-      expect(getTransformedImagePath('@test.gif')).toBe('[Image @test.gif]');
-      expect(getTransformedImagePath('@test.webp')).toBe('[Image @test.webp]');
-      expect(getTransformedImagePath('@test.svg')).toBe('[Image @test.svg]');
-      expect(getTransformedImagePath('@test.bmp')).toBe('[Image @test.bmp]');
+      expect(getTransformedImagePath('@test.jpg')).toBe('[Image test.jpg]');
+      expect(getTransformedImagePath('@test.jpeg')).toBe('[Image test.jpeg]');
+      expect(getTransformedImagePath('@test.gif')).toBe('[Image test.gif]');
+      expect(getTransformedImagePath('@test.webp')).toBe('[Image test.webp]');
+      expect(getTransformedImagePath('@test.svg')).toBe('[Image test.svg]');
+      expect(getTransformedImagePath('@test.bmp')).toBe('[Image test.bmp]');
+    });
+
+    it('should handle POSIX-style forward-slash paths on any platform', () => {
+      const input = 'C:/Users/foo/screenshots/image@2x.png';
+      expect(getTransformedImagePath(input)).toBe('[Image image@2x.png]');
+    });
+
+    it('should handle Windows-style backslash paths on any platform', () => {
+      const input = 'C:\\Users\\foo\\screenshots\\image@2x.png';
```

Comment:
are these really testing what it looks like they are testing? would expect the `@` should be at the beginning  not at the end of the path